pkgs/by-name/br/brave/make-brave.nix

```
@@ -219,6 +219,8 @@ stdenv.mkDerivation {
       # Fix paths
       substituteInPlace $out/share/applications/brave-browser.desktop \
           --replace /usr/bin/brave-browser-stable $out/bin/brave
+      substituteInPlace $out/share/applications/com.brave.Browser.desktop \
+          --replace /usr/bin/brave-browser-stable $out/bin/brave
```

```suggestion
      substituteInPlace $out/share/applications/{brave-browser,com.brave.Browser}.desktop \
          --replace /usr/bin/brave-browser-stable $out/bin/brave
```

Also please change these to `--replace-fail` while you are at it. `--replace` is deprecated.